docs/config/apache2-site

```diff
@@ -16,13 +16,13 @@
 
 	php_value include_path .:/usr/share/php:/usr/share/pear:%LORISROOT%project/libraries:%LORISROOT%php/libraries
 
-	ErrorLog /var/log/apache2/%PROJECTNAME%-error.log
+	ErrorLog /var/log/%LOGDIRECTORY%/%PROJECTNAME%-error.log
```

Comment:
This isn't consistent with the CustomLog line below it. I don't think it's supposed to have the var/log part here.